configure.ac

```
@@ -1799,37 +1811,34 @@ AS_CASE([$arch,$system],
     AC_MSG_RESULT([yes])],
   [AC_MSG_RESULT([no])])
 
-## Determine if the POSIX threads library is supported
+## Determine how to link with the POSIX threads library
+
+AS_CASE([$host],
+  [*-*-mingw32|*-pc-windows],
+    [PTHREAD_LIBS="-lpthread"],
+  [AX_PTHREAD(
+    [common_cflags="$common_cflags $PTHREAD_CFLAGS"
+    saved_CFLAGS="$CFLAGS"
+    saved_LIBS="$LIBS"
+    CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
+    LIBS="$LIBS $PTHREAD_LIBS"
+    AC_CHECK_FUNC([sigwait], [AC_DEFINE([HAS_SIGWAIT])])
+    LIBS="$saved_LIBS"
+    CFLAGS="$saved_CFLAGS"],
+    [AC_MSG_ERROR([POSIX threads are required but not supported on this platform])])])
```

To please check-typo (the generated `configure` file is unaffected by this change):
```suggestion
    [AC_MSG_ERROR(m4_normalize([POSIX threads are required but not supported on
      this platform]))])])
```

---

Thanks for the hint!  (`m4_normalize`, indeed.)  Merged and squashed with the associated commit.